### DISCUSSION TITLE
Branches are created but never merged with automergeType: branch

### DISCUSSION THREAD

**Discussion by msallent** [renovatebot/renovate#9318] [Request Help] [answered]
Hello all. I've been having this issue for quite a while already.

I have automergeType set to branch for minor and patch versions and even though branches get created, they almost never get merged.

This is my config file:

```
{
  "extends": ["config:base", "schedule:weekly"],
  "dependencyDashboard": true,
  "labels": ["dependencies"],
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch"],
      "matchCurrentVersion": "!/^0/",
      "automergeType": "branch",
      "automerge": true
    }
  ]
}
```
Before using this schedule, I had `on Mondays before 10am` where eventually 1 or 2 branches did get merged, but never all of them.

I assume that due to the schedule, Renovate is able to create branches, but never gets to actually merge them because there's no time left.

What's the expected schedule for this to work correctly 100% of the time?

**Comment by rarkins** [comment, marked as answer]
Renovate can only merge 1 branch per run, which typically means 1 branch every 2+ hours, because:
- It takes one run to create branch 1
- On the second run, it will automerge branch 1 if it's passing
- If other branches existed for auto merging, they'll need rebasing now (we never automerge branches which are not up-to-date with the main branch)
- On the run after that it auto merges

Try configuring more than one day perhaps?

**Reply by msallent** [reply]
Makes sense, thanks for the quick reply.

I'll change the schedule to be `every weekend` which I think should be more than enough time for the bot to update everything.

If thinks work out, I'll come back and mark your reply as the answer. Thanks!

**Reply by msallent** [reply]
As seen [here](https://github.com/msallent/gatsby-starter-skeleton/commits/main), the new schedule worked great. Dependencies were update one each every 2/3 hours.
